client/my-sites/site-settings/section-general.jsx

```diff
@@ -1,15 +1,27 @@
+import config from '@automattic/calypso-config';
```

Comment:
Nit: you could also import just `isEnabled` like `import { isEnabled } from ...`.